testing/N_00014_0002/test_N_00014_0002/test_output.yml

```diff
@@ -0,0 +1,69 @@
+CCD Structure Factors:
+  Final Structure Factor:
+    Momentum Transfer Vector Magnitude:
+      max: 3.872983346207417
+      min: 1.4142135623730951
+    Sum_G( S(G)*V(G) ): -0.28950840744962714
+  Initial Structure Factor:
+    Momentum Transfer Vector Magnitude:
+      max: 3.872983346207417
+      min: 1.4142135623730951
+    Sum_G( S(G)*V(G) ): -0.22730872953589673
+Eigenvalues:
+  Final Iteration:
+    MP2 Energy: -0.113654364768
+    Number of Eigenvalues: 20
+    SCF Energy: 8.925974392432
+  Initial Iteration:
+    MP2 Energy: -0.1317174123
+    Number of Eigenvalues: 20
+    SCF Energy: 9.07638714305
+HF Structure Factors:
+  Final Structure Factor:
+    Sum( ExSf*ExV ): 2.2039252419454867
+  Initial Structure Factor:
+    Sum( ExSf*ExV ): 2.089222812996651
+MP2 Structure Factors:
+  Final Structure Factor:
+    Momentum Transfer Vector Magnitude:
+      max: 3.872983346207417
+      min: 1.4142135623730951
+    Sum_G( S(G)*V(G) ): -0.22730872953589673
+  Initial Structure Factor:
+    Momentum Transfer Vector Magnitude:
+      max: 3.872983346207417
+      min: 1.4142135623730951
+    Sum_G( S(G)*V(G) ): -0.26343482459995327
+Summary:
+  Biggest CCD Energy Change: 0.0216580871
+  CCD Correlation Energy: -0.144754203725
+  Final CCD Energy: 8.781220188707
+  Iteration of Biggest CCD Energy Change: 1.0
+  Max CCD Residual: 3.830395700621e-08
+  Total Iterations: 35
+Twist Angles:
+  Averages:
+    CCD: -0.10580159883023821
+    HF: 8.166604308828415
+    MP2: -0.10580159883023821
+    ntwist x: -0.0003880977630615234
+    ntwist y: -0.040601496696472165
+    ntwist z: -0.05734013080596924
+  Connectivity Averages:
+    max: 105.4
+    min: 2.88
+  Eigenvalue Averages:
+    max: 3.4267278010364652
+    min: -0.7729035188002452
+  Errors:
+    CCD: 0.0038978450798829003
```

Comment:
With the last commits I was easily able to run the test code, however I get failures for mismatching energies, which are not actual failures because the disagreement is beyond the location for convergence.

Can we assign a precision to compare for the test calculations? 
This will need to be calculation dependent, in several ways.
The code automatically sets different convergence criteria for different rs values for CCD, as well the precision for CCD will be different for HF.
In the case of HF its trickier because the precision will be dependent on Fortran intricacies, but a broad 1e-12 should work in most cases for HF I would think.

So we will need calculation dependent parameters for HF and CCD? Not sure where MP2 falls, probably under CCD precision works fine.

Reply:
fort.80 and fort.81 quantities can use ccd precision for comparison, then everything else should be fine with 1e-12?

Reply:
I updated this PR by cherry-picking a commit from my OpenMP work that sets a general precision of 1e-13 based on the variance I saw with threading (that is, rounding numbers to 13 digits). Python's rounding function behaves deterministically so I don't see much difference between rounding and truncating for the purposes of regression testing.

I agree it makes sense to set some tolerances based on the iteration criteria. Is this rs value the `rSMin` and `rSMax` in the `Input` files? If so could you explain why there's a range?